src/annotations/markups/measurement.js

```diff
@@ -0,0 +1,73 @@
+import Enums from "../../enums";
+import { getUnitSuffix } from "../../utils";
+import {
+  getFeatureScoord3dArea,
+  getFeatureScoord3dLength,
+} from "../../scoord3dUtils.js";
+
+/**
+ * Format measure output.
+ * 
+ * @param {Feature} feature feature
+ * @param {string} units units
+ * @return {string} The formatted measure of this feature
+ */
+export const format = (feature, units, pyramid) => {
```

Comment:
I noticed two issues with the measurements markup for polygons:

1. The value doesn't seem to be correct (see comparison with resolution bar)
2. The unit is `µm`, but it should be `µm2` (or `mm2`?) in case of area

![Screen Shot 2021-04-30 at 8 24 39 AM](https://user-images.githubusercontent.com/6855297/116695671-eb1e1700-a98e-11eb-960d-c0c7ba482979.png)

Reply:
Good question, i'm not sure if we should use the units of the current resolution or mm for all resolutions.

Reply:
For https://github.com/MGHComputationalPathology/dicom-microscopy-viewer/pull/43#discussion_r623850897, can we create a separate issue? or its a blocker?

Reply:
@hackermd now hide and show should work.